packages/-ember-data/tests/integration/identifiers/lid-reflection-test.ts

```diff
@@ -145,4 +145,79 @@ module('Integration | Identifiers - lid reflection', function(hooks) {
 
     assert.strictEqual(record.name, '@runspired', 'After we finish we use the most recent clean name');
   });
+
+  test('lid works for sideposting scenario', async function(assert) {
```

Comment:
Could we beef up this test description?  Maybe something to do with "updating the client record after save" or something?

Reply:
Updated the test name